jams/sonify.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# CREATED:2015-12-12 18:20:37 by Brian McFee <brian.mcfee@nyu.edu>
+r'''
+Sonification
+============
+
+.. autosummary::
+    :toctree: generated/
+
+    sonify
+'''
+
+import six
+import numpy as np
+import mir_eval.sonify
+from mir_eval.util import filter_kwargs
+from .eval import coerce_annotation
+from .exceptions import NamespaceError
+
+__all__ = ['sonify']
+
+
+
+def clicks(annotation, sr=22050, length=None, **kwargs):
+    '''Sonify clicks timings
+
+    '''
+
+    interval, _ = annotation.data.to_interval_values()
+
+    return filter_kwargs(mir_eval.sonify.clicks, interval[:, 0],
+                         fs=sr, length=length, **kwargs)
+
+
+def chord(annotation, sr=22050, length=None, **kwargs):
+    '''Sonify chords'''
+
+    intervals, chords = annotation.data.to_interval_values()
+
+    return filter_kwargs(mir_eval.sonify.chords,
+                         chords, intervals,
+                         fs=sr, length=length,
+                         **kwargs)
+
+
+def pitch_hz(annotation, sr=22050, length=None, **kwargs):
```

Comment:
> I guess this should be fixed within jams, not mir_eval, since mir_eval doesn't claim to support negative/zero-f inputs.

By "support" do you mean in general or the sonify module? `mir_eval` does support negative/zero frequency input for evaluation (e.g. the melody module).

> Since I don't have your data handy, can you paste a from-scratch example that triggers this?

You do, it's in the jams-data repo ;)